syntax = "proto3";

import "google/protobuf/empty.proto";

package apilogger;

option java_package = "com.proto.apilogger";
option java_multiple_files = true;

service ApiLogger {
  
  //send logs
  rpc sendSingleLog(SingleLog) returns (google.protobuf.Empty){
  }

  rpc sendBatchLog(BatchLog) returns (google.protobuf.Empty){ 
  }

}

message SingleLog {
  bytes log_data = 1;
}

message BatchLog { 
  repeated bytes log_batch = 1;
}
